lib/Application.php

```diff
@@ -141,6 +141,7 @@ public static function getContainer(string $env) : ContainerBuilder
         $container->setParameter('doctrine.website.config_dir', realpath(__DIR__ . '/../config'));
         $container->setParameter('doctrine.website.cache_dir', realpath(__DIR__ . '/../cache'));
         $container->setParameter('doctrine.website.github.http_token', getenv('doctrine_website_github_http_token'));
+        $container->setParameter('doctrine.website.mysql.password', getenv('doctrine_website_mysql_password'));
```

Comment:
Environment variables can be used with `%env(...)%` in a later PR where the component can be introduced.